app/presenters/spina/pages/menu_presenter.rb

```diff
@@ -60,8 +60,9 @@ def render_menu_items(menu_items)
 
       def render_menu_item(menu_item, index, menu_items_length)
         content_tag(list_item_tag, class: menu_item_css(menu_item[0], index, menu_items_length)) do
+          path = File.join(Spina::Engine.routes.url_helpers.root_path, menu_item[0].materialized_path)
```

Comment:
A simple .join would suffice in my opinion. We shouldn't have duplicated slashes anyway.